nsfrb/searching.py

```diff
@@ -495,95 +738,46 @@ def read_cands(fname):
-    for i in range(len(cands_gaussian)):
-        if ~np.isinf(cands_gaussian[i][4]):
-            cand_ras.append(cands_gaussian[i][0])
-            cand_decs.append(cands_gaussian[i][1])
-            cand_wids.append(cands_gaussian[i][2])
-            cand_dms.append(cands_gaussian[i][3])
-            cand_snrs.append(cands_gaussian[i][4])
-    cand_ras = np.array(cand_ras)
-    cand_decs = np.array(cand_decs)
-    cand_wids = np.array(cand_wids)
-    cand_dms = np.array(cand_dms)
-    cand_snrs = np.array(cand_snrs)
-
-    cluster_cand_ras = []
-    cluster_cand_decs = []
-    cluster_cand_wids = []
-    cluster_cand_dms = []
-    cluster_cand_snrs = []
-    for i in range(len(cluster_cands_gaussian)):
-        if ~np.isinf(cluster_cands_gaussian[i][4]):
-            cluster_cand_ras.append(cluster_cands_gaussian[i][0])
-            cluster_cand_decs.append(cluster_cands_gaussian[i][1])
-            cluster_cand_wids.append(cluster_cands_gaussian[i][3])
-            cluster_cand_dms.append(cluster_cands_gaussian[i][2])
-            cluster_cand_snrs.append(cluster_cands_gaussian[i][4])
-    cluster_cand_ras = np.array(cluster_cand_ras)
-    cluster_cand_decs = np.array(cluster_cand_decs)
-    cluster_cand_wids = np.array(cluster_cand_wids)
-    cluster_cand_dms = np.array(cluster_cand_dms)
-    cluster_cand_snrs = np.array(cluster_cand_snrs)
-
-
-    plt.figure(figsize=(12,12))
-    if ~np.all(injected_cands_gaussian==None):
-        plt.plot(injected_dms,injected_wids*tsamp,'o',markersize=25,alpha=0.5,color='blue',label='injected')
-    plt.plot(DM_trials[cand_dms],widthtrials[cand_wids]*tsamp,'o',markersize=15,alpha=0.5,color='red',label='candidates')
-    plt.plot(DM_trials[cluster_cand_dms],widthtrials[cluster_cand_wids]*tsamp,'o',markersize=5,alpha=0.5,color='green',label='clusters')
-    plt.ylabel("Width (ms)")
-    plt.xlabel("DM (pc/cc)")
-    plt.legend(loc="upper left")
-    for w in widthtrials:
-        plt.axhline(tsamp*w)
-    for dm in DM_trials:
-        plt.axvline(dm)
-    plt.savefig(output_dir + "diagnostic_DM_width.png")
-    plt.close()
-
-    plt.figure(figsize=(12,12))
-    if ~np.all(injected_cands_gaussian==None):
-        plt.scatter(RA_axis[injected_ras],DEC_axis[injected_decs],marker='o',s=400,alpha=injected_snrs/np.max(injected_snrs),c='blue',label='injected')
-    plt.scatter(RA_axis[cand_ras],DEC_axis[cand_decs],marker='o',s=150,alpha=cand_snrs/np.max(cand_snrs),c='red',label='candidates')
-    plt.scatter(RA_axis[cluster_cand_ras],DEC_axis[cluster_cand_decs],marker='o',s=50,alpha=cluster_cand_snrs/np.max(cluster_cand_snrs),c='green',label='clusters')
-    plt.ylabel("RA (deg)")
-    plt.xlabel("DEC (deg)")
-    plt.legend(loc="upper left")
-    for ra in RA_axis:
-        plt.axhline(ra)
-    for dec in DEC_axis:
-        plt.axvline(dec)
+def search_plots_new(canddict,img,RA_axis=RA_axis,DEC_axis=DEC_axis,DM_trials=DM_trials,widthtrials=widthtrials,output_dir=output_dir,show=True,vmax=1000,vmin=0,s100=100):
```

Comment:
@mbsherma Move this to nsfrb/plotting.py

Reply:
Resolved in commit [78cb0ed](https://github.com/dsa110/dsa110-nsfrb/pull/10/commits/78cb0edfc85ffdf445089667f71141e579d956f9)